nemo/core/config/schedulers.py

```diff
@@ -0,0 +1,156 @@
+# =============================================================================
```

Comment:
please don't put #===  in licence header - just copy-paste our licence from dev guide

Reply:
Ah I think I copied this template all over. I'll find and replace.

Reply:
I think we should submit another PR - theres around 15 files with the above type of copyright. For now, this file has been corrected.